### PR TITLE
feat: add masking to cvc element, conform to mask on insert text

### PR DESCRIPTION
**Pull request by lcschy**
## Description

- Add masking to cvc element
- Adds conformation to mask on `insertText`

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: BasisTheoryElements/Sources/BasisTheoryElements/BasisTheoryElements.swift [hunk 1]
```diff
@@ -72,7 +72,7 @@ final public class BasisTheoryElements {
             
             let createTokenRequest = mutableBody.toCreateTokenRequest()
             
-            TokensAPI.createWithRequestBuilder(createTokenRequest: createTokenRequest).addHeader(name: "BT-API-KEY", value: getApiKey(apiKey)).execute { result in
+            TokensAPI.createWithRequestBuilder(createTokenRequest: createTokenRequest).addHeader(name: "User-Agent", value: "BasisTheory iOS Elements").addHeader(name: "BT-API-KEY", value: getApiKey(apiKey)).execute { result in
```
Comment (brigonzalez) [inline BasisTheoryElements/Sources/BasisTheoryElements/BasisTheoryElements.swift:75]: can we add this to `tokenize` as well?

File: BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeUITextField.swift [hunk 2]
```diff
@@ -23,13 +43,81 @@ final public class CardVerificationCodeUITextField: TextElementUITextField {
...
+    private func getCvcElementEvent(text: String?, event: ElementEvent) -> ElementEvent {
+        var complete = false
+        
+        if (cardNumberUITextField != nil) {
+            complete = text?.count == inputMask?.count
+        } else {
+            complete = text?.count == 3 || text?.count == 4
+        }
+        
+        let valid = self.validation?(text) ?? false
```
Comment (brigonzalez) [inline BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeUITextField.swift:68]: nitpick: you could use the `event` param's `valid` flag and just pass that back. you actually only need to change `complete` on the `event` param

File: BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeUITextField.swift [hunk 3]
```diff
@@ -23,13 +43,81 @@ final public class CardVerificationCodeUITextField: TextElementUITextField {
...
+            regexDigit
+        ]
+    }
+    
+    public func setConfig(options: CardVerificationCodeOptions) {
+        if (options.cardNumberUITextField != nil) {
+            self.cardNumberUITextField = options.cardNumberUITextField
+            
+            self.cardNumberUITextField!.subject.sink { completion in
+                //
```
Comment (brigonzalez) [inline BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeUITextField.swift:90]: is this needed?

File: BasisTheoryElements/Sources/BasisTheoryElements/BasisTheoryElements.swift (hunk 1 above)
Comment (lcschy) [reply BasisTheoryElements/Sources/BasisTheoryElements/BasisTheoryElements.swift:75]: Oopsie, forgot about it haha.

File: BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeUITextField.swift (hunk 2 above)
Comment (lcschy) [reply BasisTheoryElements/Sources/BasisTheoryElements/CardVerificationCodeUITextField.swift:68]: Good callout, changed that on the card number too.

**Review by brigonzalez — APPROVED** [review]

**Pull request merged**